app/src/main/java/com/limelight/Game.java

```diff
@@ -1133,46 +1134,43 @@ protected void onStop() {
             stopConnection();
 
             if (prefConfig.enableLatencyToast) {
-                int averageEndToEndLat = decoderRenderer.getAverageEndToEndLatency();
-                int averageDecoderLat = decoderRenderer.getAverageDecoderLatency();
-                String message = null;
-                if (averageEndToEndLat > 0) {
-                    message = getResources().getString(R.string.conn_client_latency)+" "+averageEndToEndLat+" ms";
-                    if (averageDecoderLat > 0) {
-                        message += " ("+getResources().getString(R.string.conn_client_latency_hw)+" "+averageDecoderLat+" ms)";
-                    }
-                }
-                else if (averageDecoderLat > 0) {
-                    message = getResources().getString(R.string.conn_hardware_latency)+" "+averageDecoderLat+" ms";
-                }
+                // Will format to match `\d+\.\d` (formats 1f as "1.0", 0.12f as "0.1", and 12.34f as "12.3")
+                DecimalFormat decimalFormat = new DecimalFormat("0.0");
```

Comment:
Will this do the right thing for locales that use other separators (such as `,`) for decimal values?

Reply:
Great catch. I had assumed it would inherit the same local as `getResources().getString()`, but that may be wrong according to [the docs](https://docs.oracle.com/javase/8/docs/api/java/text/DecimalFormat.html):

> To obtain a `NumberFormat` for a specific locale, including the default locale, call one of `NumberFormat`'s factory methods, such as `getInstance()`. In general, do not call the `DecimalFormat` constructors directly, since the `NumberFormat` factory methods may return subclasses other than `DecimalFormat`. If you need to customize the format object, do something like this:
> 
> ```
> NumberFormat f = NumberFormat.getInstance(loc);
>  if (f instanceof DecimalFormat) {
>      ((DecimalFormat) f).setDecimalSeparatorAlwaysShown(true);
>  }
> ```

Stay tuned for a fix...